desktop/renderer/remote-manager.js

```diff
@@ -228,8 +225,7 @@ export default connect(
     trackerStopTimer,
     trackerOnClose,
```

Comment:
I know! what happened to the branch you were working on?

I have a vague idea to clean it up that will be ~1pt and based off the comments I left you on that branch